packages/uploadthing/src/internal/handler.ts

```diff
@@ -253,8 +253,7 @@ export const buildRequestHandler = <
     }
 
     try {
-      const { files } = reqBody as { files: string[] };
-      // @ts-expect-error TODO: Fix this
+      const { files } = reqBody;
```

Comment:
files should still be unknown here really... we validate it further down (L260).

I'm guessing the previous `as` here is to support the destructuring but maybe just skip that?

```ts
try {
  // ...
  if (!Array.isArray(reqBody.files) || !reqBody.files.every((f) => typeof f === "string"))
        throw new Error("Need file array");
  // ✅ string[]
```

Reply:
so the `as` is unnecessary?